projects/aws/eks-a-admin-image/Makefile

```diff
@@ -6,24 +6,24 @@ HAS_LICENSES=false
 REPO_NO_CLONE=true
 IMAGE_NAMES=
 
-REPO=eks-a-snow-admin-image
+REPO=eks-a-admin-image
 REPO_OWNER=aws
 
 BUILD_TARGETS=validate
 RELEASE_TARGETS=build-raw-image
 
 PACKER_VERSION=1.8.0
 
-EKSA_VERSION?=v0.8.0
+EKSA_VERSION?=latest
 EKSA_RELEASE_MANIFEST_URL?=https://anywhere-assets.eks.amazonaws.com/releases/eks-a/manifest.yaml
 
 ARTIFACTS_UPLOAD_PATH?=$(PROJECT_PATH)
 EXPORT_AMI_BUCKET?=$(ARTIFACTS_BUCKET)
-FULL_OUTPUT_DIR=$(MAKE_ROOT)/$(OUTPUT_DIR)/ami/snow
+FULL_OUTPUT_DIR?=$(MAKE_ROOT)/$(OUTPUT_DIR)
 MANIFEST_OUTPUT?=$(FULL_OUTPUT_DIR)/manifest.json
```

Comment:
Don't we need to update this one too so it uses a different dir per image type?

Reply:
Appropriate format folder now gets appended to FULL_OUTPUT_DIR on the build target when called.